a.css

```diff
@@ -1300,4 +1379,62 @@ textarea {
   .l1 {
     margin-left: -215px;
   }
+
+  .small-window {
+    margin-top: 112px;
+    margin-left: 142px;
+    width: 1150px;
+    display: flex;
+  }
+
+  .h3window {
+    height: 40px;
+    width: 1040px;
+    left: 0;
+    top: 0;
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    text-align: left;
+    color: #172b4d;
+    margin-left: 0;
+  }
+
+  .awindow {
+    margin-top: 40px;
+  }
+
+  .ulwindow {
+    margin-top: 0;
+  }
+
+  .img2-window {
+    margin-left: -10px;
+    margin-top: 50px;
+  }
+
+  .p2window {
+    height: 336px;
+    width: 424px;
+    left: 0;
+    top: 0;
+    font-family: 'Inter', serif;
+    font-size: 16px;
+    font-weight: 400;
```

Comment:
- [ ] I see that you are using a transparent background for your pop up window it should be white as in the Figma design so please try to fix that 

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167954806-71fd1a60-4e4a-4170-b332-0f02e8d0e7db.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167954854-ace2cf68-ffb6-4af6-ba0f-d64de043b5b1.png)